travis_install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Script for installing various dependencies for Travis jobs.
+# Usage: ./travis_install.sh libmongoc|mongodb|swiftlint
+
+if [[ $1 == "libmongoc" ]]
+then
+	git clone -b r1.13 https://github.com/mongodb/mongo-c-driver /tmp/libmongoc
```

Comment:
I vote for making `r1.13` an environment variable here, so that we can easily switch which version of libmongoc we're testing against